source/js/components/nsf-nav.js

```diff
@@ -32,9 +32,14 @@ var MenuLinks = React.createClass({
   },
   onScroll: function() {
     var links = this.state.links || [];
+    var hash = window.location.hash.replace(`#`, ``);
     var active = links[0] || ``;
     var scrollY = window.scrollY;
 
+    if (hash && !document.querySelector(`.nav-anchor#` + hash)) {
```

Comment:
We don't have to land this. I feel like you have issues with it on a fundamental level and not something technical.

You should talk to Kristina. It's not my opinion that the accordion hash has more priority.